coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -13,6 +18,14 @@
       attach="#todo-list"
     />
 
+    <CancelPaymentErrorDialog
+      :dialog="cancelPaymentErrorDialog"
+      :errors="cancelPaymentErrors"
+      @okay="resetCancelPaymentErrors"
+      attach="#todo-list"
+      data-test-id="cancel-pay-error-dialog"
```

Comment:
In this case we don't need `data-test-id` because there is a single instance of this dialog in this component so we can already uniquely select it.